data-raw/rebuild_logo.py

```
@@ -0,0 +1,115 @@
+        f'rsvg::rsvg_png("{svg.as_posix()}","{png.as_posix()}",'
+        f'width={W*scale},height={H*scale})'], capture_output=True, text=True)
+    if r.returncode: raise RuntimeError(r.stderr[-400:])
+    frame = Image.open(png).convert("RGBA")
+
+    art = Image.open(OUT / "content.png").convert("RGBA")
+    # the artwork sat at (66,193)-(451,406) in the 518x600 original; keep that
+    # placement so the rebuild is a drop-in replacement
+    tw, th = int(art.width * scale), int(art.height * scale)
+    art = art.resize((tw, th), Image.LANCZOS)
+    frame.alpha_composite(art, (int(66 * scale), int(193 * scale)))
+    out = OUT / (f"logo.png" if scale == 1 else f"logo-{scale}x.png")
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)